apps-rendering/src/components/Dateline/index.tsx

```diff
@@ -75,18 +78,14 @@ const getDatelineStyles = (format: ArticleFormat): SerializedStyles => {
 	}
 };
 
-const Dateline: FC<Props> = ({ date, format }) =>
+const Dateline: FC<Props> = ({ date, format, edition }) =>
 	pipe(
 		date,
 		map((d) => (
-			<time
-				css={getDatelineStyles(format)}
-				data-date={d}
-				className="date js-date"
-			>
+			<time css={getDatelineStyles(format)}>
 				{format.design === ArticleDesign.LiveBlog ||
 				format.design === ArticleDesign.DeadBlog
-					? fullyFormatDate(d)
+					? datetimeFormat(edition ?? Edition.UK)(d)
```

Comment:
again, this ideally would always have a value but it's optional for now so we don't have to pass an edition everywhere yet